src/Widgets/DataFormContainerWidget/utils/isAttributeNameUnique.ts

```diff
@@ -0,0 +1,23 @@
+import { Widget } from 'scrivito'
+import { getDataFormContainer } from './getDataFormContainer'
+
+export function isAttributeNameUnique(widget: Widget) {
+  const attributeName =
+    typeof widget.get('attributeName') === 'string'
+      ? widget.get('attributeName')
+      : ''
+  if (!attributeName) return true
```

Comment:
I suggest the following: 

```suggestion
  const attributeName = widget.get('attributeName')
  if (typeof widget.get('attributeName') !== 'string') return true
  if (!attributeName) return true
```